### ISSUE TITLE
Flux Network Energy

### ISSUE THREAD

**Issue by SwissPegasus** [Wilyicaro/BetterFurnacesReforged#143]
### Better Furnaces Reforged Version

1.20.1-1.1.1.forge

### Mod Loader Version

Forge 47.2.0

### Bug/Glitch Occurrence Context

Flux Energy and Extrem Furnace Energy transfer

### What happened wrong?

Using Flux Energy (Flux Point) to get energy inside extrem furnace from behind (try also all other sides). This works perfect under 1.19.2 but not on the 1.20.1.
On my 1.19.2 installation is use the same version of your mod.

For shure i use the same upgrades, facotry upgrade and energy upgrade.

**Comment by SwissPegasus** [comment]
I verify the working on 1.19.2, also not working anymore. On the older version this worked.

**Comment by SwissPegasus** [comment]
Hi, i don't know which older Version of your mod worked as last. In some of the newer release it must stop working. In Flux Point there is no way to force output or i don't know how...

**Comment by Wilyicaro** [comment]
I tested it with the Flux Networks mod  in the same 1.20.1 version you tested (1.1.1) and it's working correctly, so I think you made a mistake when configuring the Flux Point or something similar.

**Comment by SwissPegasus** [comment]
Hi, but what i need to configure. Just Connect on the backside and thats all, select the network u like to take the enery from and finish.

**Comment by SwissPegasus** [comment]
Another test, i installed a mod with energy pipes. They do not connect to your furnace (extreme furnace). So there is no other way to get energy in. 
![2023-12-01_21 46 38](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/4289952/c353d0f2-99df-4676-9bfe-2426ec44a4d4)

**Comment by Wilyicaro** [comment]
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/1474fbd5-b874-4fc9-818a-a55f8da7118c)
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/54e30c1b-427d-4f82-b02c-4a0370e275de)
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/1237bc9f-1d25-4c99-9481-2301b925c572)

**Comment by Wilyicaro** [comment]
Thanks for helping with this list even you at work, but unfortunately I installed every mod from it and this conflict didn't happen for some reason.
I hope you forgot the mod that is causing this conflict.
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/304ed881-9b59-4dc1-9b47-7c0625b3ac48)
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/fc7af5c1-544b-4229-be32-d5afc079bcf1)
![image](https://github.com/Wilyicaro/BetterFurnacesReforged/assets/81383021/4f893736-fb46-446d-a3cc-52ed9651418c)

**Comment by Wilyicaro** [comment]
Yes, you are right, I made the mistake of replacing the Multipart Machine: Grinding by the cooking one, but even with the right mod everything is working correctly in my test, unfortunately I didn't discovered the problem

**Comment by SwissPegasus** [comment]
so, what i have to change?

**Comment by Wilyicaro** [comment]
If you can, remove the Optifine, there are many alternatives for its features

**Comment by Wilyicaro** [comment]
This a Optifine bug, it probably will be fixed in the future.
https://github.com/sp614x/optifine/issues/7549#issue-1896464826
